.github/workflows/push_ocm.yaml

```diff
@@ -70,3 +67,67 @@ jobs:
             ghcr.io/open-component-model/ocm/ocm.software/ocmcli/ocmcli-image:latest
             ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  ocm-cli-latest:
+    name: ocm-cli build latest
+    runs-on: large_runner
+    permissions:
+      contents: write
+      id-token: write
+      packages: write
+    steps:
+    - name: Self Hosted Runner Post Job Cleanup Action
+      uses: TooMuch4U/actions-clean@v2.2
+    - name: Generate token
+      id: generate_token
+      uses: tibdex/github-app-token@v2
+      with:
+        app_id: ${{ secrets.OCMBOT_APP_ID }}
+        private_key: ${{ secrets.OCMBOT_PRIV_KEY }}
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: Enable gh-CLI
+      uses: actions4gh/setup-gh@v1.0.2
+      with:
+        token: ${{ steps.generate_token.outputs.token }}
+    - name: delete release and move tag
+      run: |
+        if [ "$(gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/open-component-model/ocm/releases | grep -c 'ocm-cli/latest')" -ge 1 ]; then
```

Comment:
Where does this date `2022-11-28` come from? Do we have to update this in the future?

Reply:
I guess this is the Api-Version for GitHub. However, this seems an odd identifier for the API imo.^^

Reply:
sorry, but it's regular github stuff - https://docs.github.com/de/rest/about-the-rest-api/api-versions